docs/reference/changelog.mdx

```diff
@@ -6,6 +6,16 @@ toc_max_heading_level: 3
 
 # Changelog
 
+## Nov 1, 22024
+
+### Kafka Client (0.31.0)
+
+✨ [responsive-pub/370](https://github.com/responsivedev/responsive-pub/pull/370):
```

Comment:
obviously this was actually a set of PRs not just the one, but I picked the first since that was where we introduced the API. Doesn't seem useful to list out every single individual commit